package.json

```diff
@@ -20,5 +20,9 @@
   "bugs": {
     "url": "https://github.com/umap-project/Leaflet.Measurable/issues"
   },
-  "homepage": "https://github.com/umap-project/Leaflet.Measurable#readme"
+  "homepage": "https://github.com/umap-project/Leaflet.Measurable#readme",
+  "dependencies": {
```

Comment:
I think it's better to warn about dependencies in the README instead.
We had a bunch of troubles when setting non stable dependencies in plugins.